manual/embedded-examples/src/docs/dev/procedures.asciidoc

```diff
@@ -69,15 +69,15 @@ Neo4j comes with a handful of pre-packaged procedures ready to be used:
 
 [options="header", cols="m,m,d"]
 |===
-| Procedure name        | Command to invoke procedure | What it does
-| ListLabels            | CALL db.labels()            | List all labels in the database.
-| ListRelationshipTypes | CALL db.relationshipTypes() | List all relationship types in the database.
-| ListPropertyKeys      | CALL db.propertyKeys()      | List all property keys in the database.
-| ListIndexes           | CALL db.indexes()           | List all indexes in the database.
-| ListConstraints       | CALL db.constraints()       | List all constraints in the database.
-| ListProcedures        | CALL sys.procedures()       | List all procedures in the DBMS.
-| ListComponents        | CALL sys.components()       | List DBMS components and their versions.
-| QueryJmx              | CALL sys.queryJmx(query)    | Query JMX management data by domain and name. For instance, "org.neo4j:*".
+| Procedure name        | Command to invoke procedure  | What it does
+| ListLabels            | CALL db.labels()             | List all labels in the database.
+| ListRelationshipTypes | CALL db.relationshipTypes()  | List all relationship types in the database.
+| ListPropertyKeys      | CALL db.propertyKeys()       | List all property keys in the database.
+| ListIndexes           | CALL db.indexes()            | List all indexes in the database.
+| ListConstraints       | CALL db.constraints()        | List all constraints in the database.
+| ListProcedures        | CALL dbms.procedures()       | List all procedures in the DBMS.
+| ListComponents        | CALL dbms.components()       | List DBMS components and their versions.
+| QueryJmx              | CALL dbms.queryJmx(query)    | Query JMX management data by domain and name. For instance, "org.neo4j:*".
```

Comment:
Because procedures are per dbms under the current deployment model while labels might be database in a multi-tenancy world.